syslinux-native: fix parallel building issue

There might be an error when parallel build:

[snip]
cp: cannot create directory `tmp/sysroots/x86_64-linux/usr/share/
syslinux/com32/include/gplinclude': No such file or directory
make[4]: *** [install] Error 1
make[3]: *** [gpllib] Error 2
[snip]

This is a potential issue. In ${S}/com32/gpllib/Makefile file,
install target wants to copy $(SRC)/../gplinclude to
$(INSTALLROOT)$(COM32DIR)/include/ directory, but in ${S}/com32/lib/Makefile
file, the install target will remove $(INSTALLROOT)$(COM32DIR)/include
directory. We need to do com32/lib first.

The patch make com32/gpllib depends on com32/lib to fix this issue.

Upstream-Status: Pending

Signed-off-by: Chong Lu <Chong.Lu@windriver.com>
---
 com32/Makefile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/com32/Makefile b/com32/Makefile
index 7ea1b01..9a1721b 100644
--- a/com32/Makefile
+++ b/com32/Makefile
@@ -20,3 +20,4 @@ modules: lib libutil gpllib
 rosh: lib libutil
 samples: libutil elflink/ldlinux
 sysdump: libupload gpllib
+gpllib: lib
-- 
1.7.9.5

